Assets/GothicVR/Scripts/UI/TextureManager.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using UnityEngine;
+using GVR.Caches;
+using GVR.Util;
+using GVR.Phoenix.Interface;
+using TMPro;
+public class TextureManager : MonoBehaviour
+{
+    private Texture2D backgroundtexture;
+    public Material backgroundmaterial;
```

Comment:
As Frank mentioned: This will be replaced with a never option from his scene. I ask you to create a ticket for "Refacturing materials, remove and load them dynamically like LoadingMaterial". To be in sync. Once ticket is created, you can close this comment.

Reply:
kk.